.github/workflows/container.yml

```diff
@@ -1,7 +1,7 @@
 name: Container
 on:
     schedule:
-        -  cron: '30 11 * * *'   # every day at 4:40
+        -  cron: '30 11 * * *'   # every day at 11:30 GMT
```

Comment:
GMT or UTC?

Reply:
You are right @aafeijoo-suse . It should be UTC. I changed it.
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule